### PR TITLE
Add random props

### PR DESCRIPTION
**Pull request by setoelkahfi**

### CODE REVIEW

**Comment by charj** [comment]
I wanted this the other day :+1:

**Comment by adrianmcli** [comment]
@charj thanks, I'll revisit this.

**Comment by setoelkahfi** [comment]
> I wanted this the other day 👍

I almost forget that I made this PR 😀

File: src/ReactRotatingText.js [hunk 1]
```diff
@@ -89,12 +90,17 @@ class ReactRotatingText extends React.Component {
...
   _animate() {
     const { index } = this.state;
-    const { items, pause, emptyPause, eraseMode } = this.props;
+    const { items, pause, emptyPause, eraseMode, random } = this.props;
     const type = this._type;
     const erase = this._erase;
     const overwrite = this._overwrite;
     const loopingFunc = this._animate.bind(this);
-    const nextIndex = index === items.length - 1 ? 0 : index + 1;
+    var nextIndex;
```
Comment (adrianmcli) [inline src/ReactRotatingText.js]: Can you change this to use `let` instead?

Comment (setoelkahfi) [reply src/ReactRotatingText.js]: done

**Comment by adrianmcli** [comment]
Published!

**Pull request merged**